### ISSUE TITLE
Admin panel 50% width when following a row with multiple fields

### ISSUE THREAD

**Issue by scisteffan** [springload/wagtailgmaps#37]
See the issue here:

![screenshot from 2018-06-02 17-30-34](https://user-images.githubusercontent.com/2676185/40878550-b2d46ec0-668a-11e8-9820-ece88151955e.png)

I fixed this by adding `clear: both` to the li.wagtailgmap CSS style.

**Comment by loicteixeira** [comment]
Thanks for the report @scisteffan. Would you might submitting a PR with the fix?

**Comment by scisteffan** [comment]
Sure, I've submitted my first ever PR!

**Comment by loicteixeira** [comment]
FWIW, testing shows that it only happens when the last children of the `FieldRowPanel` is a `FieldPanel` for a `date` field (but works fine for a text field`).

```
content_panels = Page.content_panels + [
    FieldRowPanel(
        [
            FieldPanel('expire_at'),
            FieldPanel('go_live_at'),
        ]
    ),
    MapFieldPanel('formatted_address'),
]
```

**Comment by loicteixeira** [comment]
It should be available on PyPi as 1.0.1 now. Thanks again.
